.github/workflows/build-images.yml

```diff
@@ -1,94 +1,59 @@
 name: Build images
 
 on:
+  # does not work with forks since the jobs will try to push to the target
+  # registry but run only with the privileges of the fork.
+  #- pull_request
   - push
 
 jobs:
-  cc7_lcg95:
+  base-image:
     runs-on: ubuntu-latest
-    needs: base_images
-    steps:
-      - uses: actions/checkout@v1
-      - name: Extract branch name
-        shell: bash
-        run: echo "::set-env name=branch::$(echo ${GITHUB_REF#refs/heads/} | sed 's/\//_/g')"
-      - uses: acts-project/Publish-Docker-Github-Action@master
-        with:
-          registry: gitlab-registry.cern.ch
-          name: acts/machines/cc7_lcg95
-          username: ${{ secrets.DOCKER_USERNAME  }}
-          password: ${{ secrets.DOCKER_PASSWORD  }}
-          context: cc7_lcg95
-          buildargs: branch
-
-  cc7_lcg96:
-    runs-on: ubuntu-latest
-    needs: base_images
-    steps:
-      - uses: actions/checkout@v1
-      - name: Extract branch name
-        shell: bash
-        run: echo "::set-env name=branch::$(echo ${GITHUB_REF#refs/heads/} | sed 's/\//_/g')"
-      - uses: acts-project/Publish-Docker-Github-Action@master
-        with:
-          registry: gitlab-registry.cern.ch
-          name: acts/machines/cc7_lcg96
-          username: ${{ secrets.DOCKER_USERNAME  }}
-          password: ${{ secrets.DOCKER_PASSWORD  }}
-          context: cc7_lcg96
-          buildargs: branch
-
-  slc6_lcg95:
-    runs-on: ubuntu-latest
-    needs: base_images
-    steps:
-      - uses: actions/checkout@v1
-      - name: Extract branch name
-        shell: bash
-        run: echo "::set-env name=branch::$(echo ${GITHUB_REF#refs/heads/} | sed 's/\//_/g')"
-      - uses: acts-project/Publish-Docker-Github-Action@master
-        with:
-          registry: gitlab-registry.cern.ch
-          name: acts/machines/slc6_lcg95
-          username: ${{ secrets.DOCKER_USERNAME  }}
-          password: ${{ secrets.DOCKER_PASSWORD  }}
-          context: slc6_lcg95
-          buildargs: branch
-
-  cc7_clang_tidy:
-    runs-on: ubuntu-latest
-    needs: base_images
+    strategy:
+      matrix:
+        directory:
+          - centos7
+          - check-format
+          - fedora31
+          - slc6
```

Comment:
So no ubuntu build anymore?

Reply:
Temporarily, yes. The existing Ubuntu 19.10 build does not satisfy the Boost version requirement (1.69 required, 1.67 available) and the 20.04 build still had some issues. The files are there and should be reactivated in May after 20.04 has been released.

Reply:
But we run coverage in ubuntu1910...

Reply:
Also, it currenlty runs in the CI in the main repo. I don't see what the problem is honestly.

Reply:
The examples require Boost 1.69 and the ubuntu1910 build does not provide it. I do not want to manually install Boost and initially wanted to switch to ubuntu 20.04 as the base instead. I could not get it to work. I had better luck using a Fedora 31-based image. This (temporarily) replaces the ubuntu images in the monorepo PR both for regular builds and for the coverage.

Reply:
The CI in the core reports boost `1.72` at cmake time.

Reply:
The Ubuntu 20.04 build with (almost) all dependencies is now available again and replaces the Fedora 31 image. I left the Dockerfile for the Fedora 31 image for future reference.